bb.edn

```diff
@@ -18,13 +23,22 @@
                      :task (clojure "-M:test:sci:demo:dev watch browser")
                      :depends [yarn-install]}
 
-         release:js {:doc "Builds & uploads the js release and updates the sha reference."
-                     :task (do (clojure "-M:sci:demo:dev release browser")
-                               (clojure "-T:build upload-to-cas+rewrite-sha :resource viewer.js"))
-                     :depends [yarn-install]}
+         build:js {:doc "Builds JS"
+                   :depends [yarn-install]
+                   :task (clojure "-M:sci:demo:dev release browser")}
+
+         write-hash {:doc "Write hash to resources. Intended to be used in a pre-commit hook."
+                     :task hashing/write-hash}
+
+         refresh-cdn {:doc "Refreshes assets stored on CDN (google storage)"
```

Comment:
It's not clear to me from the name or doc what this tasks does. Does it build the viewer.js and upload it to google cloud storage?

Reply:
Yes